src/functionalTest/java/uk/gov/hmcts/sptribs/testutil/ServiceAuthenticationGenerator.java

```diff
@@ -19,9 +19,16 @@ public class ServiceAuthenticationGenerator {
     @Value("${s2s.name}")
     private String s2sName;
 
+    @Value("${s2s.task_management_name}")
+    private String s2sTMName;
```

Comment:
For readability is it worth going more verbose with this variable name? eg. s2sTaskManagementName?